agent/src/lib.rs

```diff
@@ -18,6 +18,9 @@ impl Agent {
             content: text.to_string(),
         })
     }
+    pub fn to_string(&self) -> String {
```

Comment:
Please insert an empty line above this one.

Reply:
done